ethcore/builtin/Cargo.toml

```diff
@@ -14,7 +14,7 @@ ethereum-types = "0.9.0"
 ethjson = { path = "../../json" }
 keccak-hash = "0.5.0"
 log = "0.4"
-num = { version = "0.1", default-features = false, features = ["bigint"] }
+num = "0.2"
```

Comment:
we still only use `BigInt` from `num` so why enable the default features?

Reply:
`bigint` is not a feature anymore - `num` is an export of multiple crates now. I could reduce the dependency to `num-bigint` though.

Reply:
Ok, I see. It is hidden behind the `std flag` don't bother then...